snuba/query/joins/subquery_generator.py

```diff
@@ -58,7 +58,9 @@ def build_query(self) -> ProcessableQuery[Entity]:
 
 
 def aliasify_column(col_name: str) -> str:
-    return f"_snuba_{col_name}"
+    from snuba.query.parser import ALIAS_PREFIX
```

Comment:
Why not importing this at the top of the file? There is noo need to go tthrough the import process each time

Reply:
I had to do local dynamic import due to circular import issues when I put the import at the top of the file.

Reply:
The fact it causes a circular import is a sign that something is not right in the code organization, we should try to fix that instead of hacking around it. 
What was the module/symbol that was causing the circular import and can we move it in a different place ?

Reply:
I have removed the refactoring from this PR so this problem is not part of the PR anymore.